packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -108,13 +108,9 @@ function ChangelogTooltip<TRef extends HTMLElement>({
             data-testid="changelog"
           >
             <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
-              <Button
-                disabled
-                className="font-normal text-white bg-theme-color-water btn-primary small"
-                data-testid="changelogNewReleaseTag"
-              >
+              <h3 className="ml-2 font-bold typo-title3 text-theme-label-primary">
                 New release
```

Comment:
I see that it now says `New version` in sketch. Just to check if text should also change.

Reply:
Should be "New release"
[relevant discussion](https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1682240390128199?thread_ts=1681963065.569039&cid=C01L0QXQJNB).